pkg/database/vercode.go

```diff
@@ -313,6 +315,21 @@ func (db *Database) DeleteVerificationCode(code string) error {
 		Error
 }
 
+// RecycleVerificationCodes sets to null code and long_code values
+// so that status can be retained longer, but the codes are recycled into the pool.
+func (db *Database) RecycleVerificationCodes(maxAge time.Duration) (int64, error) {
+	if maxAge > 0 {
+		maxAge = -1 * maxAge
+	}
+	deleteBefore := time.Now().UTC().Add(maxAge)
+	// Null out the codes where this can be done.
+	rtn := db.db.Model(&VerificationCode{}).
+		Select("code", "long_code").
+		Where("expires_at < ? AND long_expires_at < ? AND code IS NOT NULL", deleteBefore, deleteBefore).
```

Comment:
Should we add a check about long_code here too?

Reply:
done, with OR